### ISSUE TITLE
Customize context parameter to include build name

### ISSUE THREAD

**Issue by amir-arad** [jonnyzzz/TeamCity.GitHub#89]
It is now possible to display results of multiple CI builds and systems by using different values in the context field of the report.

This will greatly improve visibility of branch / PR status if more than one job is concerned.

**Comment by griwes** [comment]
... #86, #87, #88 ...

**Comment by sihil** [comment]
:+1:

**Comment by sihil** [comment]
Yes - it would make a lot of sense for the default to be the name of the TeamCity build, which is pretty much what I'm laboriously writing in to the content field by hand.

**Comment by thevery** [comment]
Is it possible to set branch name as context parameter? Tried %build.branch% android a bunch of other parameters, but without any success, unfortunately :(

**Comment by thevery** [comment]
@iOSCowboy can you help me please with problem above as the author of context support feature?

**Comment by thevery** [comment]
Aaah, fixed via configuration instead of system/env parameter.

**Comment by jonathanadler** [comment]
@thevery I know this is old, but do you perhaps remember what exactly you did to fix this?

**Comment by thevery** [comment]
AFAIR instead of 2 status reports like 'feature_branch' and 'pull_request_number/merge' I've switched to 2 build configs from template, BUT better solution is to use built-in commit status publisher plugin which resolves this issue out of the box.
